### ISSUE TITLE
Voiceover issues

### ISSUE THREAD

**Issue by phillipthelen** [HabitRPG/habitrpg-ios#504]
You can currently expand the checklist, but then not navigate and interact with the items.

**Comment by phillipthelen** [comment]
Additional issues from an email

1.  Many of the buttons in tavern  are being labeled as party instead of whatever they should be labeled as.
2. I couldnt dismiss or  most “Justin” messages and tutorial screens with VO enabled.
3. I find many of the text blocks dont quite get focus properly. Sometimes they  something beneath a modal gets focus, and sometimes only part of a message is read and i have difficulty focusing the rest  to finish reading it.
4.  I don’t have a nonvisual way to tell what color my tasks are. With habits and dailys can that info be read after the streak number?
